src/telegram-types.ts

```diff
@@ -99,6 +99,7 @@ export type ExtraVenue = MakeExtra<
 export type ExtraVideo = MakeExtra<'sendVideo', 'video'>
 export type ExtraVideoNote = MakeExtra<'sendVideoNote', 'video_note'>
 export type ExtraVoice = MakeExtra<'sendVoice', 'voice'>
+export type ExtraBanChatSenderChat = MakeExtra<'banChatSenderChat'>
```

Comment:
I'm not sure why there's no `ExtraBanChatMember`. I also don't know if `sender_chat` should be passed here.

Reply:
### Bot API 5.3

* Renamed the method `kickChatMember` to `banChatMember`. The old method name can still be used.

That's why. It's `ExtraKickChatMember`.

Reply:
I've deprecated `kickChatMember` and introduced `banChatMember`. Although this change is from Bot API 5.3, and I'd originally assumed it to be previously complete.